PSPrintSDK/OLPosterViewController.m

```diff
@@ -62,10 +61,12 @@ -(void)viewDidLoad{
     self.imageViews = [[NSMutableArray alloc] initWithCapacity:self.product.quantityToFulfillOrder];
     for (NSUInteger i = 0; i < self.product.quantityToFulfillOrder; i++){
```

Comment:
While it is redundant right now, we will add the different poster templates soon. Should we take it out until then?